src/Marten/Events/Aggregation/EventSlicer.cs

```diff
@@ -104,11 +104,27 @@ internal IEnumerable<Type> DetermineEventTypes()
     /// <param name="mode">Should the fan out operation happen after grouping, or before? Default is after</param>
     /// <typeparam name="TEvent"></typeparam>
     /// <typeparam name="TChild"></typeparam>
-    public EventSlicer<TDoc, TId> FanOut<TEvent, TChild>(Func<TEvent, IEnumerable<TChild>> fanOutFunc,
-        FanoutMode mode = FanoutMode.AfterGrouping)
+    public EventSlicer<TDoc, TId> FanOut<TEvent, TChild>(Func<TEvent, IEnumerable<TChild>> fanOutFunc, FanoutMode mode = FanoutMode.AfterGrouping)
```

Comment:
Gotta add an overload rather than just using the optional argument 'cause semantic versioning. Irritating, but them's the rules

Reply:
Same as above, GitHub started misbehaving and showed diff wrongly.

Reply:
I think it's non-breaking now